docs/decisions/install-extensions-for-existing-lazy-loaded-feature/install-extensions-for-existing-lazy-loaded-feature.md

```diff
@@ -0,0 +1,395 @@
+## 1. Title
```

Comment:
I'm happy to add more details to this doc, if it contributes to the decision, context, or pros/cons. At the same time I'd like to balance the level of details to be eaisly digestable for the reader.

If some info impacts the decision, I'd like to copy/rephrase it directly in this doc, instead of linking to external resources (like ticket). I'd like this document to be standalone, comprehensive and immutable (in terms of git commit history).

Reply:
Ok, that makes sense. 
I was thinking we could link it like: for the current docs, "please visit this page", or "see this ticket/PR".